tests/test_minhash.py

```diff
@@ -559,6 +559,14 @@ def test_similarity_1(track_abundance):
     assert round(b.similarity(b), 3) == 1.0
 
 
+def test_copy(track_abundance):
```

Comment:
question - do `FrozenMinHash` objects work properly with `copy()` in this way, too?

Note you can make one with `MinHash.to_frozen(...)` to check.